i18n/en-US.properties

```diff
@@ -22,8 +22,14 @@ be.activitySidebar.activityFeed.showReplies = See {repliesToLoadCount, plural, o
 be.activitySidebar.activityFeed.taskMissingError = This task no longer exists
 # One of the dropdown options to select in order to filter comments
 be.activitySidebarFilter.status.all = All Comments
+# One of the dropdown options to select in order to filter activities
+be.activitySidebarFilter.status.allActivity = All Activity
 # One of the dropdown options to select in order to filter comments
 be.activitySidebarFilter.status.open = Unresolved Comments
+# One of the dropdown options to select in order to filter comments
+be.activitySidebarFilter.status.resolved = Resolved Comments
+# One of the dropdown options to select in order to filter activities
+be.activitySidebarFilter.status.task = Tasks
```

Comment:
Could be something like `One of the dropdown options to select in order to filter tasks`. 

And I think we can make this plural`be.activitySidebarFilter.status.tasks`

Reply:
Should we include the specific filter name in the comment? i.e. `The all activity dropdown option to select for filtering activities`